src/backupNg/backupNg.spec.js

```diff
@@ -66,4 +66,42 @@ describe("backupNg", () => {
       expect(schedule.jobId).toBe(jobId);
     });
   });
+
+  describe(".delete() :", () => {
+    it("deletes a backup job", async () => {
+      let scheduleId = "scheduleTempId";
+      const { id: jobId } = await xo.call("backupNg.createJob", {
+        ...defaultBackupNg,
+        schedules: {
+          [scheduleId]: {
+            name: "scheduleTest",
+            cron: "0 * * * * *",
+          },
+        },
+        settings: {
+          ...defaultBackupNg.settings,
+          [scheduleId]: { snapshotRetention: 1 },
+        },
+      });
+      const backupNgJob = await xo.call("backupNg.getJob", { id: jobId });
+      for (const key in backupNgJob.settings) {
+        if (key !== "") {
+          expect(backupNgJob.settings[key]).toEqual({ snapshotRetention: 1 });
+          scheduleId = key;
+        }
+      }
+
+      await xo.call("backupNg.deleteJob", { id: jobId });
+      await Promise.all([
+        expect(
+          xo.call("backupNg.getJob", { id: jobId })
+        ).rejects.toMatchInlineSnapshot(
```

Comment:
What is the behavior of this test if we decide to use another code formatter than prettier ? And why use it instead of `toMatchSnapshot`?

Reply:
Excuse me, I didn't pay attention. I wanted to use `toMatchSnapshot`.

Reply:
I cannot use `toMatchSnapshot` because the error will change each time because of the job ID. But you're right if we use another code formatter, it doesn't work. I will try to find a better solution.

Reply:
`toMatchInlineSnapshot` has the same behaviour than  `toMatchSnapshot` [except the snapshot is storred in the current source file](https://jestjs.io/docs/en/expect#tomatchsnapshotpropertymatchers-snapshotname).

Reply:
If you want to test precisely an error message, `toMatchSnapshot` will not be necessary. You can use `toBe`.